### PR TITLE
Add option to reset scene audio player when navigating to scene

### PR DESCRIPTION
**Pull request by boyum**

### CODE REVIEW

File: scripts/components/HUD/Buttons/AudioButton.js [hunk 1]
```diff
@@ -171,6 +171,17 @@ export default class AudioButton extends React.Component {
...
+
+      const isNewScene = this.props.sceneId !== prevProps.sceneId;
+      if (this.props.restartAudioOnSceneStart && isNewScene) {
+        const currentPlayerId = this.getPlayerId(this.props);
+        const player = this.getPlayer(currentPlayerId);
+  
+        player.audioTrack = 0;
+        player.currentTime = 0;
+  
+        console.log({player})
```
Comment (henriettemoe) [inline scripts/components/HUD/Buttons/AudioButton.js]: Remove console.log here?

File: language/nb.json [hunk 2]
```diff
@@ -198,8 +201,8 @@
       ]
     },
     {
-      "label": "Playlists",
-      "description": "These options will let you add playlists for the scenes.",
+      "label": "Spillelister",
+      "description": "Disse valgene lar deg legge til spillelister i scenene.",
       "fields": [
         {
           "label": "Added playlists",
```
Comment (henriettemoe) [inline language/nb.json]: "label": "Spillelister lagt til" ?

```suggestion
          "label": "Spillelister lagt til",
```

File: scripts/components/HUD/Buttons/AudioButton.js (hunk 1 above)
Comment (boyum) [reply scripts/components/HUD/Buttons/AudioButton.js]: Thanks!

File: language/ca.json [hunk 3]
```diff
@@ -271,6 +266,130 @@
     },
     {
       "label": "Localitza",
+      "description": "These options will let you control how the world behaves.",
```
Comment (henriettemoe) [inline language/ca.json]: Is this correct?

File: language/ca.json [hunk 4]
```diff
@@ -203,15 +206,7 @@
       "fields": [
         {
           "label": "Audio type",
-          "description": "Decide what audio type this scene will use, either use an audio track or choose an existing audio playlist.",
-          "options": [
-            {
-              "label": "Global Audio"
```
Comment (henriettemoe) [inline language/ca.json]: Should this be removed?

File: language/ca.json (hunk 3 above)
Comment (boyum) [reply language/ca.json]: No 😢

File: language/nb.json [hunk 5]
```diff
@@ -225,19 +228,19 @@
...
         {
-          "label": "Display \"Scores\" button",
-          "description": "Display button for showing the scores from assignments."
+          "label": "Vis \"Poeng\"-knappen",
+          "description": "Vis en knapp som lar brukeren se antall poeng som er scoret."
         },
         {
           "label": "Kvalitet på scene-rendering",
-          "description": "Choose the amount of width and height segments used to render a scene. This directly affects the quality level of the scene, try increasing the quality if a scene looks \"blocky\" or \"waves\" are seen within the scenes. Note that higher quality rendering takes more time to load.",
+          "description": "Velg Choose the amount of width and height segments used to render a scene. This directly affects the quality level of the scene, try increasing the quality if a scene looks \"blocky\" or \"waves\" are seen within the scenes. Note that higher quality rendering takes more time to load.",
```
Comment (boyum) [inline language/nb.json]:
```suggestion
          "description": "Choose the amount of width and height segments used to render a scene. This directly affects the quality level of the scene, try increasing the quality if a scene looks \"blocky\" or \"waves\" are seen within the scenes. Note that higher quality rendering takes more time to load.",
```

**Review by henriettemoe — APPROVED** [review]

**Pull request merged**